src/main/java/uk/gov/companieshouse/AppealApplication.java

```diff
@@ -2,13 +2,12 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
 
-@EnableMongoRepositories
 @SpringBootApplication
 public class AppealApplication {
 
     public static void main (String[] args) {
         SpringApplication.run(AppealApplication.class, args);
     }
+
```

Comment:
can we remove that new line? Dont think it aids visualization in this case as not much is going on in terms of code